homeassistant/components/stookwijzer/__init__.py

```diff
@@ -2,29 +2,68 @@
 
 from __future__ import annotations
 
+import logging
+
 from stookwijzer import Stookwijzer
 
 from homeassistant.config_entries import ConfigEntry
 from homeassistant.const import CONF_LATITUDE, CONF_LOCATION, CONF_LONGITUDE, Platform
 from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryError
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
 
 from .const import DOMAIN
+from .coordinator import StookwijzerCoordinator
 
 PLATFORMS = [Platform.SENSOR]
 
+_LOGGER = logging.getLogger(__name__)
+
+type StookwijzerConfigEntry = ConfigEntry[StookwijzerCoordinator]
+
 
 async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
```

Comment:
```suggestion
async def async_setup_entry(hass: HomeAssistant, entry: StookwijzerConfigEntry) -> bool:
```

Reply:
Not needed, as there is no need to be specific in this case... but 🤷